pkg/storage/ec/client_planet_test.go

```diff
@@ -0,0 +1,172 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package ecclient_test
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/vivint/infectious"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/pkg/auth/signing"
+	"storj.io/storj/pkg/eestream"
+	"storj.io/storj/pkg/pb"
+	ecclient "storj.io/storj/pkg/storage/ec"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite"
+	"storj.io/storj/storagenode"
+)
+
+const (
+	dataSize     = 32 * memory.KiB
+	storageNodes = 4
+)
+
+func TestECClient(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	planet, err := testplanet.New(t, 1, storageNodes, 1)
+	if !assert.NoError(t, err) {
```

Comment:
you can use `require.NoError(t, err)` to fail fast

Reply:
True. Missed this one.